pricegraph/src/orderbook.rs

```diff
@@ -192,6 +192,10 @@ impl Orderbook {
             });
         }
 
+        for orders in &mut [&mut overlap.asks, &mut overlap.bids] {
+            orders.sort_unstable_by(|a, b| num::compare(a.price(), b.price()));
+        }
```

Comment:
If we do this we should probably mention it in the api documentation.

Reply:
I removed it in the next PR. It turns out ordering first was a bad idea. I will just remove it here as well.